python_modules/dagster/dagster/_core/storage/legacy_storage.py

```diff
@@ -444,7 +479,8 @@ def get_asset_events(
         before_timestamp=None,
         cursor: Optional[int] = None,  # deprecated
     ) -> Union[Iterable["EventLogEntry"], Iterable[Tuple[int, "EventLogEntry"]]]:
-        return self._storage.event_log_storage.get_asset_events(
+        # Type ignore bc method does not exist-- unexecuted code?
```

Comment:
This approach is excellent. Use the pragma to get it passing but document the knowledge you accrued along that way as a comment 👍🏻

Reply:
This is the approach I used where the fix wasn't obvious